pandas/_libs/tslibs/timezones.pyx

```diff
@@ -45,7 +41,7 @@ from pandas._libs.tslibs.util cimport (
 
 cdef int64_t NPY_NAT = get_nat()
 cdef tzinfo utc_stdlib = timezone.utc
-cdef tzinfo utc_pytz = pytz.utc
+cdef tzinfo utc_pytz = None
```

Comment:
Shouldn't this be `pytz.utc if pytz else None`?

Reply:
Good call. Changed